### PR TITLE
Add organization profile

### PR DESCRIPTION
**Pull request by wooorm**
### Initial checklist

*   [x] I read the support docs 
*   [x] I read the contributing guide 
*   [x] I agree to follow the code of conduct 
*   [x] I searched issues and couldn’t find anything (or linked relevant results below) 
*   [x] If applicable, I’ve added docs and tests

### Description of changes

[View rendered](https://github.com/remarkjs/.github/blob/profile/profile/readme.md).

I want to add a profile for the remark org. It will be [rendered here](https://github.com/remarkjs?type=source).
This file contains that. I already pushed an initial version so it can be more easily previewed.
But in this PR I want to discuss what should end up there and amend it.

### CODE REVIEW

**Comment by wooorm** [comment]
There aren’t a lot of examples of orgs with profiles yet:

- https://github.com/microsoft
- https://github.com/twitter
- https://github.com/github

**Review by Murderlon — COMMENTED** [review]
Nice, didn't know this existed for orgs. Might be good to mention remark falls under the unified collective?

**Comment by wooorm** [comment]
I was hoping the:

```md
*   for more about us, see `unifiedjs.com`
*   for updates, see Twitter
 ```
 
…of which the latter links to unifiedjs, not explain that enough? What would you add?

**Comment by Murderlon** [comment]
When '[Crowdsourcing the evolution of text parsing with unified](https://medium.com/unifiedjs/collectively-evolving-through-crowdsourcing-22c359ea95cc)' was written we defined the goal of the collective as:

> [...] an effort to bring together like-minded organisations to collaboratively work on the innovation of content through seamless, interchangeable, and extendible tooling. We build parsers, transformers, and utilities so that others don’t have to worry about syntax

So it could perhaps be interesting to have a bit of that shared vision in the org profiles, and optionally link to the other orgs part of it, or just unified, and optionally only link from there to the other orgs.

**Comment by wooorm** [comment]
@Murderlon how about this?

**Review by Murderlon — APPROVED** [review]

**Comment by wooorm** [comment]
Not bad, right? [view](https://github.com/remarkjs?type=source)

**Comment by ChristianMurphy** [comment]
I think I like it?
It's frustrating that it pushes repositories [below the fold](https://www.wordstream.com/below-the-fold).
But it provides some good context.
Screenshot 2021-11-17 at 10-33-07 remark

**Comment by wooorm** [comment]
We could put unified and get involved in a details: “More info”.
But I dunno. It’s different for sure but I’m not sure it’s worse?

**Comment by ChristianMurphy** [comment]
> We could put unified and get involved in a details: “More info”.

That's one option.

Another would be cutting the full width logo

Screenshot 2021-11-17 at 10-47-50 remark

**Comment by wooorm** [comment]
yeah. I personally quite like it. It grabs the viewer. (if limited data says anything, the inspiration also all have it: https://github.com/remarkjs/.github/pull/5#issuecomment-971431479)

**Comment by wooorm** [comment]
I think explaining what remark is, and linking to what’s most important, in prose, might be more important than showing projects as fast as possible?

File: profile/readme.md [hunk 1]
```diff
@@ -1,26 +1,69 @@
...
+
+[twitter]: https://twitter.com/unifiedjs
+
+[collective]: https://github.com/unifiedjs/collective
+
+[coc]: ../code-of-conduct.md
+
+[support]: ../support.md
+
+[contribute]: ../contributing.md
```
Comment (ChristianMurphy) [inline profile/readme.md:65]: these links lead to 404 not found

Comment (wooorm) [reply profile/readme.md:65]: lol. While writing this I was like: hmm, interesting how GH resolves markdown, must be pretty smart.
And then again, it wasn’t smart...

Comment (wooorm) [reply profile/readme.md:65]: solved!

**Comment by ChristianMurphy** [comment]
> yeah. I personally quite like it. It grabs the viewer

Maybe something still flashy but with less padding around the edges?

Screenshot 2021-11-17 at 11-11-50 remark

**Comment by ChristianMurphy** [comment]
And if some of the bullet points could be turned horizontal. (maybe tables), even more space could be saved, while still conveying the key information.

**Comment by wooorm** [comment]
But what's the real problem of hiding pinned projects under the fold? I get that it's a change, but there are some very useful links in there, that might be better to have above the fold?

**Comment by ChristianMurphy** [comment]
> But what's the real problem of hiding pinned projects under the fold?

Because people want to get to the repositories.

This hides that with the options after the change being:
* scroll 6-8 clicks to either find the pinned repo or search the repo name
* click "repositories" wait for a full page refresh, then search for the repo you want
* Use the top search to find in organization, then switch to repository tab

I'm not saying your content is bad, it's good!
I'm saying whoever had the idea that:

* a header
* followed by a hero
* followed by another header
* followed by another hero
* before people get to what the are most likely actually looking for _the repos_

may not have thought through that design fully. :sweat_smile:

**Comment by wooorm** [comment]
> Because people want to get to the repositories.

Do they? the list of repositories is mostly garbage: last updated, not very useful. The pinned repos are a bit useful. But the intro already links to half of these, and links to three good places to find plugins. Those explain plugins in more detail, and include more plugins (not all plugins are maintained in remark).
The top search bar by default searches for repos in an org for me? I get search. But I think that already does it?

> before people get to what the are most likely actually looking for the repos

Is that really the goal of this page? It was the only thing on org pages before. So again I get that it’s a change. But I have a hunch that there might be other valid reasons for people to visit the org page: to learn about the org and find the most useful links (not just repos).

I don’t think many folks really go and search for plugins by searching or browsing our org, instead I’m assuming they use global github search or our lists of plugins?

**Comment by ChristianMurphy** [comment]
> Is that really the goal of this page?

I really don't know GitHub's goal for the page.
The uses for it seem like a vary from: about us, to contact us, to a myspace page.

> It was the only thing on org pages before. So again I get that it’s a change. But I have a hunch that there might be other valid reasons for people to visit the org page: to learn about the org and find the most useful links (not just repos).

That could be, and if so, putting as much of that information that we think people want/need to see in the fold would still be a goal.

**Comment by wooorm** [comment]
I meant why people go to that page. I’m not sure what you mean by MySpace page, but it sounds like we agree what should go on the org page? About, contact, profile. Not top repos / all repos?

> That could be, and if so, putting as much of that information that we think people want/need to see in the fold would still be a goal.

I’m not sure what you propose here? There is a lot of useful information in there?

---

Is the problem the repos being too low? Or is the image a problem?

#### Repos higher

If there is a problem with repos being too low, we can try and get them higher:

Screen Shot 2021-11-18 at 10 55 16

I think this hides too much useful information behind details, on the GitHub app it puts useful information behind two interactions, I personally don’t get why repos should be as high as possible, and as folds are dynamic it wouldn’t work everywhere.

#### No image

If the image is too large, we can remove it:

Screen Shot 2021-11-18 at 10 55 46

I’m 🤷‍♂️ on this. It does make more info available, not the repos though, it feels rather busy to me.

**Pull request merged**